tc/sgx/common/packages/base64/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Intel Corporation
+# Copyright 2020 Intel Corporation
```

Comment:
Not a related change for SGX error handling. But fine as it's a minor change updating copyright year.